EMGD's gl2.h is missing the GLchar typedef, so add it.

Upstream-Status: Submitted
Signed-off-by: Ross Burton <ross.burton@intel.com>

diff --git a/usr/include/GLES2/gl2.h b/usr/include/GLES2/gl2.h
index 8b83f3f..4fbd12e 100644
--- a/usr/include/GLES2/gl2.h
+++ b/usr/include/GLES2/gl2.h
@@ -21,6 +21,7 @@ typedef unsigned int    GLenum;
 typedef unsigned char   GLboolean;
 typedef unsigned int    GLbitfield;
 typedef khronos_int8_t   GLbyte;
+typedef char            GLchar;
 typedef short           GLshort;
 typedef int             GLint;
 typedef int             GLsizei;
